manifests/bind_event.pp

```diff
@@ -0,0 +1,34 @@
+# Define: qpid::bind_event
+#
+# This define binds the event queue in katello with the correct messages from candlepin to filter out uneeded messages
```

Comment:
Drop references to Katello and Candlepin throughout. Try to think of `puppet-qpid` as a module that knows how to configure Qpid and that other modules can configure it to their needs. So you want to strive for making and referencing generic parameters and concepts specific to qpid alone.